app/services/account_getter_test.go

```diff
@@ -0,0 +1,53 @@
+package services_test
+
+import (
+	"sort"
+	"testing"
+
+	"github.com/keratin/authn-server/app/data/mock"
+	"github.com/keratin/authn-server/app/services"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAccountGetter(t *testing.T) {
+	t.Run("returns empty map when no oauth accounts", func(t *testing.T) {
```

Comment:
Hopefully not something we'll see at runtime but its probably worth adding a test for not found and the case where the oauth account lookup errors while we are here.